### DISCUSSION TITLE
Mixture-of-Agents Large Language Model Capabilities local with Ollama/MLStudio

### DISCUSSION THREAD

**Discussion by Kingbadger3d** [open-webui/open-webui#3187] [Ideas]
Hey Dudes, Ive come across a very impressive MOA agent example that has been hacked into running local by a user within hours. This shouldf be able to plug into the openai api endpoint openwebui already supports with some tweaks.

Github Repo: https://github.com/bolecki/MoA/tree/local-test

Dev, Any chance you can take a look and start adding this support to your tool? Agents are the way forward and the reason why I started playing with your GUI tool knowing agents would be a best use case in future. This is a big step forward and your tool could be top of the line with MOA's added.

Cheers. KB

**Comment by GeekyGhost** [comment]
From the papers on it, it seems you can merge and layer LLMs to make them equal to or better than GPT 4. Man, I thought you couldn't merge or layer LLMs like that? Definitely would be nice to have that ability in open WebUI. 

On a serious note, it really would be nice if Open WebUI had it. 😅
